src/vmod_geoip.c

```diff
@@ -62,14 +62,13 @@ vmod_region_name_by_addr(GeoIP *gi, const char *ip)
 		const char *str = NULL;					\
 									\
 		CHECK_OBJ_NOTNULL(ctx, VRT_CTX_MAGIC);			\
-		CHECK_OBJ_NOTNULL(ctx->ws, WS_MAGIC);			\
 		AN(pp->priv);						\
 									\
 		if (ip)							\
 			str = func(pp->priv, ip);			\
 		if (str == NULL)					\
 			return ("Unknown");				\
```

Comment:
<bikeshedding>str = "Unknown"; would allow for a single return statement and make me happy</bikeshedding>